terraform/groups/lambda/module-lambda/variables.tf

```diff
@@ -6,7 +6,11 @@ variable timeout_seconds {}
 variable service {}
```

Comment:
These should have types as a minimum and ideally descriptions too

Reply:
Done. Added types to all variables and descriptions to variables at the point where they're defined. I've avoided repeating descriptions where variables are passed into submodules and have the same names.